features/gezag-minderjarige/1.1-ingezetene-BRP.feature

```diff
@@ -0,0 +1,87 @@
+  gezagsmutatie wordt bijvoorbeeld niet bijgewerkt in de RNI.
+
+  Een persoon is "ingezetene" en "ingeschreven in de BRP" wanneer deze staat ingeschreven
+  in een Nederlandse gemeente. In dat geval gaat de gezagbepaling 
+  verder met vraag 1.2.
+
+  Een persoon is "niet ingezetene" wanneer deze staat ingeschreven in het RNI (Register Niet 
+  Ingezetenen). In dat gezal is het gezag niet te bepalen.
+
+
+    Achtergrond:
+      Gegeven de persoon 'Gerda' met burgerservicenummer '000000012'
+      * is meerderjarig, niet overleden en staat niet onder curatele
+      En de persoon 'Bert' met burgerservicenummer '000000024'
+      * is meerderjarig, niet overleden en staat niet onder curatele
+      En 'Gerda' en 'Bert' zijn met elkaar gehuwd
+      En de persoon 'Zoe' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is niet geëmigreerd geweest
+      * is in Nederland geboren
+      * heeft 'Gerda' als ouder 1
+      * heeft 'Bert' als ouder 2
+
+
+  Regel: Als de persoon staat ingeschreven in het RNI kan het gezag van de persoon niet worden bepaald
+
+    Scenario: minderjarige staat ingeschreven in RNI
+      Gegeven persoon 'Zoe' is ingeschreven in de RNI
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                               |
+      | type        | GezagNietTeBepalen                                                   |
+      | toelichting | gezag is niet te bepalen omdat minderjarige niet in Nederland woont. |
+
+    Scenario: minderjarige staat ingeschreven in een Nederlandse gemeente
+      Gegeven persoon 'Zoe' is ingeschreven in de BRP
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+
+
+  Regel: onderzoek op gemeente van inschrijving leidt niet tot uitval
```

Comment:
Zou hier niet de rule moeten zijn: onderzoek op de gemeente van inschrijving wordt meegeleverd met gezag?
Dat zouden we eigenlijk nog verder uit moeten werken, ook voor de werkgroep afleidingsregels.

Reply:
op dit moment beschrijf ik wat de gezag-api doet. die negeert onderzoek op gemeente van inschrijving.

het is voor de toekomt dan de vraag of we, wanneer we inOnderzoek leveren in plaats van GezagNietTeBepalen bij onderzoek, ook onderzoek op gemeente van inschrijving moeten leveren. Maar voor nu negeert de gezag-api dit dus